.github/workflows/testing.yml

```diff
@@ -0,0 +1,210 @@
+name: testing
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run_tests_linux:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.8'
+          - '2.9'
```

Comment:
I would remove tarantool-2.9 (here and everywhere else), because it was never released.

Reply:
Removed